test/bacom-blog/link/auto-link.md

```diff
@@ -0,0 +1,7 @@
+<https://www.youtube.com/watch?v=_e8XLnMiCOE>
+
++-----------------------------------------------+
+| Iframe                                        |
++===============================================+
+| <https://adobe.com/>                          |
++-----------------------------------------------+
```

Comment:
These are two different cases, so I think they should be in two different files and tests

Reply:
Fair, I'll split these and move them into a mocks folder.